### PR TITLE
Revision of tests

### PR DESCRIPTION
**Pull request by aSvetenco**

### CODE REVIEW

File: MeetUp/app/src/test/java/com/endava/meetup/screens/auth/modules/signin/SignInPresenterTest.java [hunk 1]
```diff
@@ -118,26 +118,43 @@ public void detachView_ShouldCallCancelAsyncTask() {
         Mockito.verify(model).cancelAsyncTask();
     }
 
+    @Test
+    public void onSignInButtonClick_ShouldCallDoSignIn() {
+        onEmailTextChanged_ShouldClearValidationError_WhenValidData();
```
Comment (CebotariM) [inline MeetUp/app/src/test/java/com/endava/meetup/screens/auth/modules/signin/SignInPresenterTest.java]: dont use underscore, use Java convention CamelCase

File: MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java [hunk 2]
```diff
@@ -63,9 +66,9 @@ public void setUp() throws Exception {
 
     @Test
     public void onUserGroupResponse_ShouldFillGroupsModel_WhenResponseNotEmpty() {
-        Group group = Mockito.mock(Group.class);
-        List<Group> data = new ArrayList<>();
-        data.add(group);
+        Group group = new Group();
```
Comment (CebotariM) [inline MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java]: can you mock the group?

File: MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java [hunk 3]
```diff
@@ -154,6 +152,22 @@ public void onErrorWithResponseThrowable_ShouldCallOnError() {
         Mockito.verify(callback).onError(errorHandler.handleError(throwable));
     }
 
+    @Test
+    public void tryToUseDbData_ShouldCallOnGroupsUpdate_WhenDbIsNotEmpty() {
```
Comment (CebotariM) [inline MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java]: Please use camelcase everywhere

File: MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java [hunk 4]
```diff
@@ -154,6 +152,22 @@ public void onErrorWithResponseThrowable_ShouldCallOnError() {
...
+        ModelResultCallback callback = Mockito.mock(ModelResultCallback.class);
+        Topic topic = new Topic();
+        topic.setId(DEF_TOPIC_ID);
+        Group group = new Group();
```
Comment (CebotariM) [inline MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java]: mocks?

File: MeetUp/app/src/test/java/com/endava/meetup/screens/auth/modules/signin/SignInPresenterTest.java (hunk 1 above)
Comment (aSvetenco) [reply MeetUp/app/src/test/java/com/endava/meetup/screens/auth/modules/signin/SignInPresenterTest.java]: ok, i'm going to change naming (just was thinking that is more readable with underscore)

File: MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java (hunk 2 above)
Comment (aSvetenco) [reply MeetUp/app/src/test/java/com/endava/meetup/screens/home/HomeActivityModelTest.java]: aim for using real object  - to set id 
but now I think it possible to use construction -  when(someObject.getId).thenReturn(neededId) 
i'll do changes

**Pull request merged**